controllers/managedocs_controller.go

```diff
@@ -507,6 +510,19 @@ func (r *ManagedOCSReconciler) reconcilePhases() (reconcile.Result, error) {
 			r.addonParams[key] = string(value)
 		}
 
+		if err := r.reconcileEgressNetworkPolicy(); err != nil {
+			return ctrl.Result{}, err
+		}
+		if err := r.reconcileIngressNetworkPolicy(); err != nil {
+			return ctrl.Result{}, err
+		}
+		if err := r.reconcileCephIngressNetworkPolicy(); err != nil {
+			return ctrl.Result{}, err
+		}
+		if err := r.reconcileProviderAPIServerNetworkPolicy(); err != nil {
+			return ctrl.Result{}, err
+		}
+
```

Comment:
By reconciling security related objects earlier in the stack, we can ensure that the service isn't missing any security configuration when the storage is available.

Reply:
Actually since this is breaking tests, I'll revert it.